profiles/cloudways.json

```diff
@@ -9,121 +9,303 @@
-    {
-      "continent": "AS",
-      "country": "JP",
-      "region": "Tokyo",
-      "provider": "AWS"
-    },
-    {
-      "continent": "EU",
-      "country": "IE",
-      "region": "Dublin",
-      "provider": "AWS"
-    },
-    {
-      "continent": "NA",
-      "country": "US",
-      "region": "Northern California",
-      "provider": "AWS"
-    },
-    {
-      "continent": "NA",
-      "country": "US",
-      "region": "Northern Virginia",
-      "provider": "AWS"
-    },
-    {
-      "continent": "NA",
-      "country": "US",
-      "region": "Oregon",
-      "provider": "AWS"
-    },
-    {
-      "continent": "OC",
-      "country": "AU",
-      "region": "Sydney",
-      "provider": "AWS"
-    },
-    {
-      "continent": "SA",
-      "country": "BR",
-      "region": "Sao Paulo",
-      "provider": "AWS"
-    }
-  ]
+  "infrastructures": [{
+    "continent": "EU",
+    "country": "NL",
+    "region": "Amsterdam",
+    "provider": "DigitalOcean"
+  }, {
+    "continent": "NA",
+    "country": "US",
+    "region": "San Francisco",
+    "provider": "DigitalOcean"
+  }, {
+    "continent": "NA",
+    "country": "US",
+    "region": "New York",
+    "provider": "DigitalOcean"
+  }, {
+    "continent": "AS",
+    "country": "SG",
+    "region": "Singapore",
+    "provider": "DigitalOcean"
+  }, {
+    "continent": "EU",
+    "country": "GB",
+    "region": "London",
+    "provider": "DigitalOcean"
+  }, {
+    "continent": "EU",
+    "country": "DE",
+    "region": "Frankfurt",
+    "provider": "DigitalOcean"
+  }, {
+    "continent": "AS",
+    "country": "JP",
+    "region": "Tokyo",
+    "provider": "DigitalOcean"
+  }, {
+    "continent": "AS",
+    "country": "SG",
+    "region": "Singapore",
+    "provider": "AWS"
+  }, {
+    "continent": "AS",
+    "country": "JP",
+    "region": "Tokyo",
+    "provider": "AWS"
+  }, {
+    "continent": "NA",
+    "country": "US",
+    "region": "Northern California",
+    "provider": "AWS"
+  }, {
+    "continent": "NA",
+    "country": "US",
+    "region": "Northern Virginia",
+    "provider": "AWS"
+  }, {
+    "continent": "NA",
+    "country": "US",
+    "region": "Oregon",
+    "provider": "AWS"
+  }, {
+    "continent": "OC",
+    "country": "AU",
+    "region": "Sydney",
+    "provider": "AWS"
+  }, {
+    "continent": "SA",
+    "country": "BR",
+    "region": "Sao Paulo",
+    "provider": "AWS"
+  }, {
+    "continent": "AS",
+    "country": "SK",
+    "region": "Seoul",
+    "provider": "AWS"
+  }, {
+    "continent": "EU",
+    "country": "DE",
+    "region": "Frankfurt",
+    "provider": "AWS"
+  }, {
+    "continent": "EU",
+    "country": "IE",
+    "region": "Ireland",
+    "provider": "AWS"
+  }, {
+    "continent": "GB",
+    "country": "US",
+    "region": "Iowa",
+    "provider": "GCE"
+  }, {
+    "continent": "GB",
+    "country": "US",
+    "region": "South Carolina",
+    "provider": "GCE"
+  }, {
+    "continent": "EU",
+    "country": "GB",
+    "region": "GB",
```

Comment:
The two GCE entries are not correct. Please leave region empty if it is not known or correct both entries.